packages/headless/src/controllers/facets/_common/facet-option-definitions.ts

```diff
@@ -19,10 +19,26 @@ export const basePath = new ArrayValue({
   each: new StringValue(),
 });
 
+/**
+ * The character that specifies the hierarchical dependency.
+ */
```

Comment:
Regarding defaults, these are listed in the `defaultFacetOptions` variable inside `facet-set-slice.ts`. They should be the same as what is documented for the search-api [here](https://docs.coveo.com/en/1461/build-a-search-ui/query-parameters#definitions-RestFacetRequest).